idea_town/frontend/static-src/app/views/header-view.js

```diff
@@ -30,20 +29,15 @@ export default BaseView.extend({
   //       to pick up csrftoken cookie changes.
   logout(evt) {
     evt.preventDefault();
-    xhr({
+    fetch(app.getDocHeadLink('x-logout', 'href'), {
```

Comment:
To be clear, I'm happy to merge this bit as-is, just asking discussion questions